docs/plugins.md

```diff
@@ -412,12 +412,12 @@ Content of foo is : {{ foo }}.
 {% endif %}
 ```
 
-Please note that every form submission is protected via a CSRF token, you will need to include the follow snippet into your forms :
+Please note that every form submission is protected via a CSRF token, you will need to include the following snippet into your forms :
 ```html
 <input type="hidden" name="csrf_token" value="{{ csrf_token() }}" />
 ```
 
-Retrieving user submitted data is pretty simple thanks to the request module provided by Flask :
+Retrieving user submitted data is pretty simple, thank to the request module provided by Flask :
```

Comment:
waiting confirmation but I thought "thanks to" is the right one

Reply:
Both of them can be used, which one do you prefer ?